README.md

```diff
@@ -1,8 +1,12 @@
 # fastly
 
+[![Cookbook Version](https://img.shields.io/cookbook/v/fastly.svg)](https://supermarket.chef.io/cookbooks/fastly)
+[![CI State](https://github.com/sous-chefs/fastly/workflows/ci/badge.svg)](https://github.com/sous-chefs/fastly/actions?query=workflow%3Aci)
+[![OpenCollective](https://opencollective.com/sous-chefs/backers/badge.svg)](#backers)
+[![OpenCollective](https://opencollective.com/sous-chefs/sponsors/badge.svg)](#sponsors)
+[![License](https://img.shields.io/badge/License-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
 # EXPERIMENTAL
 
 This cookbook will provide resources for working with Fastly Services.
 
 It is completly a work in progress. Please contact cwebber@chef.io with any questions.
```

Comment:
This contact should be updated and you also need to add our other usual blocks we have in the README.